pkg/kritis/metadata/metadata.go

```diff
@@ -144,3 +153,59 @@ func GetAttestationsFromOccurrence(occ *grafeas.Occurrence) ([]cryptolib.Attesta
 	}
 	return atts, nil
 }
+
+// CreateOccurrenceFromAttestation creates an occurrence from an attestation by specified signature type.
+// The created occurrence can either be a PgpSignedAttestation occurrence or a GenericSignedAttestation occurrence.
+func CreateOccurrenceFromAttestation(att *cryptolib.Attestation, containerImage string, noteName string, sType SignatureType) (*grafeas.Occurrence, error) {
+	attestation := &attestationpb.Attestation{}
```

Comment:
nil ?

Reply:
nil doesn't seem to make compiler happy :(